plugins/com.google.cloud.tools.eclipse.appengine.validation/src/com/google/cloud/tools/eclipse/appengine/validation/XmlSourceValidator.java

```diff
@@ -84,8 +84,10 @@ public void validate(IValidationContext helper, IReporter reporter) throws Valid
   /**
    * Adds an {@link IMessage} to the XML file for every
    * {@link ElementProblem} found in the file.
+   * @throws CoreException 
    */
-  void validate(IReporter reporter, IFile source, byte[] bytes) throws IOException {
+  @VisibleForTesting
+  void validate(IReporter reporter, IFile source, byte[] bytes) throws IOException, CoreException {
```

Comment:
`CoreException` should be removed, probably. The Javadoc too.

Reply:
done